ocs_ci/ocs/resources/pod.py

```
@@ -1376,11 +1375,11 @@ def check_file_existence(pod_obj, file_path):
-        check_if_executable_in_path(pod_obj.exec_cmd_on_pod("which find"))
-    except CommandFailed:
-        pod_obj.install_packages("findutils")
-    ret = pod_obj.exec_cmd_on_pod(f'bash -c "find {file_path}"')
+    # try:
+    #     check_if_executable_in_path(pod_obj.exec_cmd_on_pod("which find"))
+    # except CommandFailed:
+    #     pod_obj.install_packages("findutils")
+    ret = pod_obj.exec_cmd_on_pod(f'bash -c "ls {file_path}"')
     if re.search(file_path, ret):
         return True
     return False
```

Ruff (0.15.21)

[warning] Return the condition `bool(re.search(file_path, ret))` directly

Replace with `return bool(re.search(file_path, ret))`

(SIM103)